Contributions/Criteria-Documentation-Evaluation.md

```diff
@@ -0,0 +1,114 @@
+| Reviewers | Reviewer of the document, should have relevant experience. |
+| Date | Date of the current version of the document. Note that this can differ from the file date. |
+
+### Review Ratings (Example only)
+
+| Rating | Explanation |
+| :---- | :---- |
+| Critical  | Showstopper, the document is fundamentally incorrect in its current state and should be reworked.  Must be fixed immediately. |
+| Severe | Major problem with the document.  Must be addressed or fixed before the document can be considered for further review. |
+| Minor | Problem with the document, should be corrected, but review can continue. |
+| Observation | Comment from the reviewer to the author about how something might be improved.  Not a problem with the document. |
+| Question | Could be considered as a minor also.  This is a question from the reviewer to the author to elaborate or clarify something that is unclear.  Could also be considered as a violation of the Clear, Comprehensible, or Detail property. |
+| Not Applicable | It does not make sense to use this criterion for this evaluation. |
+
+## Document review process:
+
+Entry Conditions:
+
+* An enumeration method (e.g. headings, line numbers, etc.) should be available for ease of review document feedback references.
+* A mechanism should be available for tracking changes to a review document.
+* Author requests the review
+  * Author can use the review criteria to determine if the document is mature enough to be reviewed
+  * The Author fills out the file metadata
+  * The Author fills out the file header metadata
+* Review lead is selected from the approved list
+* Reviewers are selected by the review lead
+* Relevant documents are made available
+  * Perform review
+    * Violations should be noted as: Critical (C), Severe (S), Minor (M), Observation (O), Question (Q), Not Applicable (N)
+  * Mark feedback according to criteria
+  * Feedback should be included as comments unless the change is minor
+  * Feedback should be assigned to the author to address
+  * Note to author:
+    * The reviewers are not directing their feedback at you or your ability.  The purpose of this is to find potential bugs and minimize systematic error in the documentation
+
+Exit Condition:
+
+* Fewer remaining major defects per page than the agreed exit standard
+* Recommendation is one (1) Severe (or higher) defect per page as an initial exit criterion
+
+## Review Checklist
+
+| Review ID | Criterion | Reference | Rating | Comments |
+| :---- | :---- | :---- | :---- | :---- |
+| ID\_01 | Is the document unambiguously clear to the target audience? | Location(s) of the violation of the criterion | C, S, M, O, Q, N  | Feedback on what the problem is and a suggested correction. |
+| ID\_02 | Do the stakeholders and consumers of the document understand its meaning? |  |  |  |
+| ID\_03 | Is there sufficient detail? |  |  |  |
+| ID\_04 | Are complex concepts described as measurable elementary concepts? |  |  |  |
```

Comment:
I could see how it might be desirable to describe complex concepts in terms of concepts with which the audience might reasonably be familiar, but I'm not sure that concepts are always 'measurable'...

Reply:
Goal here is to break down the claim, investigation or argument that you are making or conducting into elementary concepts that the intended audience can understand. Perhaps "quantifiable" instead of "measurable".